velox/docs/functions/spark/map.rst

```diff
@@ -0,0 +1,34 @@
+===========================
+Map Functions
+===========================
+
+.. function:: getmapvalue(map(K,V), key) -> V
```

Comment:
Is getmapvalue a valid SparkSQL function?

Reply:
Actually, I also feel curious, some functions were registered in Register.cpp. but I can not find them in Spark Functions.
https://github.com/facebookincubator/velox/blob/main/velox/functions/sparksql/Register.cpp#L102 register a function for spark getmapvalue. but I can not find getmapvalue in  https://spark.apache.org/docs/latest/api/sql/index.html. 
 So which one should I refer to?

Reply:
It is possible that some functions were added incorrectly and need to be removed.

Also, some functions correspond to operators, hence, their names may not be searchable, e.g.  equalnullsafe corresponds to `<=>`. We need to identify these cases and document accordingly, e.g. say that function xxx corresponds to Spark's operator yyy.

Reply:
Ok, So we should refer to https://spark.apache.org/docs/latest/api/sql/index.html. I will remove unmatch functions.